pkg/gardenlet/apis/config/validation/validation.go

```diff
@@ -236,6 +240,22 @@ func ValidateManagedSeedControllerConfiguration(cfg *config.ManagedSeedControlle
 	return allErrs
 }
 
+// ValidateNetworkPolicyControllerConfiguration validates the network policy controller configuration.
+func ValidateNetworkPolicyControllerConfiguration(cfg *config.NetworkPolicyControllerConfiguration, fldPath *field.Path) field.ErrorList {
```

Comment:
```suggestion
// validateNetworkPolicyControllerConfiguration validates the network policy controller configuration.
func validateNetworkPolicyControllerConfiguration(cfg *config.NetworkPolicyControllerConfiguration, fldPath *field.Path) field.ErrorList {
```

Should we make this function private (and the [other ones](https://github.com/gardener/gardener/blob/7489574cf91eb6c5ad5f2ed3597104b7d0163145/pkg/gardenlet/apis/config/validation/validation.go#L162-L175) called in `ValidateGardenletConfiguration` as well)?